lib/api/createFile.js

```diff
@@ -114,7 +114,7 @@ module.exports = function createFile(bucketId, fileName, file, opts, cb) {
 // isn't one already
 function fileToStream(file) {
   // Is it already a readable stream?
-  if(file instanceof stream.Readable) {
+  if(file.readable) {
```

Comment:
Justification for is-stream is that it handles a lot of edge cases that we aren't covering here.